static/app/components/core/select/index.chonk.tsx

```diff
@@ -68,7 +68,7 @@ export const getChonkStylesConfig = ({
       background: theme.background,
       border: `1px solid ${theme.border}`,
       boxShadow: theme.dropShadowMedium,
-      borderRadius: theme.borderRadius,
+      borderRadius: theme.formRadius[size].borderRadius,
```

Comment:
@Jesse-Box `theme.borderRadius` is still `6px`, while we’ve changed it for forms to be:

```
md: 8px
sm: 6px
xs: 5px
```

I’m not sure where `theme.borderRadius` is used (it has a lot of usages), or if it should stay 6px or 8px, or if it should even exist ...

Reply:
agree, but is this something we should already do? It would mean most usages would go from 6px to 8px, and I’m not sure all places even have a `size` as a prop, which means we would likely also go to `md` (8px)

Reply:
Not yet, there's just too many things to update if we do that and like you said, the size props is sparse and non consistent (I'm pretty sure I've seen certain components default to different sizes).  If we can, lets avoid this for now, otherwise the sensible thing would be to backport to old theme and just set it to 6px for all sizes and roll with the new values in chonk